app/assets/javascripts/ace.js.coffee

```diff
@@ -1,4 +1 @@
-#= require ace/ace
-#= require ace/mode-javascript.js
-#= require ace/mode-markdown.js
-#= require ace/mode-coffee.js
+#= require ace-rails-ap
```

Comment:
Is it possible to only include the modes that we currently use? I noticed the asset pre-compilation step now takes significantly longer because ace is building the JavaScript for all available modes.